src/otx/core/data/entity/classification.py

```diff
@@ -68,7 +68,7 @@ def collate_fn(
         batch_data = super().collate_fn(entities)
         return MulticlassClsBatchDataEntity(
             batch_size=batch_data.batch_size,
-            images=tv_tensors.Image(data=torch.stack(batch_data.images, dim=0)),
```

Comment:
Did you check conflicts properly?

Reply:
I checked train and test with torch model. We have same interface for other tasks, I am not sure why we added torch.stack here. @sungmanc do you have comments/concerns here?

Reply:
Ok, I see where can be a problem during validation. I will apply fix accordingly

Reply:
It came from https://github.com/openvinotoolkit/training_extensions/pull/2749

Reply:
I'm not sure how it works without error. `OTXModel(nn.Module)` might raise the error since `batch_data.images` is `List[tv_tensors.Image]` while it expects the `Tensor`. After the PR what @vinnamkim pointed out, there is no `torch.stack` in the `_customized_input`.

Reply:
I will merge and take a look one more time

Reply:
OTXBatchDataEntity can receive list of tensors as well. There won't be problems. Training and validation is working
![image](https://github.com/openvinotoolkit/training_extensions/assets/68103869/355a1763-cdc6-472f-b9f8-91562208779c)

Reply:
We can't use torch stack for OV IR, since it will require resizing operation, but Model_API do it inside. We shouldn't resize it before. Other tasks are also use list[tensor] in batch

Reply:
Please, let me know if it will raise any issues. For now, it seems to work OK, integration tests prove it